### PR TITLE
Config/base package

### PR DESCRIPTION
**Pull request by krystianpach**

### CODE REVIEW

File: dist/SampleComponent.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+import { Text } from 'react-native';
+export const SampleComponent = ({ text }) => (<Text>
```
Comment (jpudysz) [inline dist/SampleComponent.js]: semicolons, indentation

File: dist/index.js [hunk 2]
```diff
@@ -0,0 +1 @@
+export { SampleComponent } from './SampleComponent';
```
Comment (jpudysz) [inline dist/index.js]: .

File: dist/SampleComponent.js (hunk 1 above)
Comment (krystianpach) [reply dist/SampleComponent.js]: this is dist folder...

File: dist/index.js (hunk 2 above)
Comment (krystianpach) [reply dist/index.js]: aa

**Pull request merged**